src/Data/Newtype.purs

```diff
@@ -17,6 +43,17 @@ ala
   -> f a
 ala _ f = map unwrap (f wrap)
 
+-- | Similar to `ala` but useful for cases where you want to use an additional
+-- | projection with the higher order function:
+-- |
+-- | ``` purescript
+-- | ala' Additive foldMap String.length ["hello", "world"] -- 10
+-- | ala' Multiplicative foldMap Math.abs [1.0, -2.0, 3.0, -4.0] -- 24.0
+-- | ```
+-- |
+-- | The type admits other possibilities due to the polymorphic `Functor`
```

Comment:
Yeah, I believe so, have you got a short example that would work? I figured it'd probably be useful with `transformers`, but I've not got to the point that builds with all the other stuff yet, so I couldn't test anything out.